src/emailpro/account/emailpro-account.popover.html

```diff
@@ -1,24 +1,21 @@
 <div data-ng-controller="EmailProToolboxAccountsCtrl" class="popover-menu">
 
-    <button class="btn btn-link"
-            type="button"
+    <button class="btn btn-link" type="button"
             data-i18n-static="exchange_tab_ACCOUNTS_popover_alias"
-            data-ng-if="!is25g();"
             data-ng-class="isDisabled(account)"
-            data-ng-click="aliasDisplay(account)">
+            data-ng-click="aliasDisplay(account)"
+            data-ng-if="!is25g()">
```

Comment:
Just so you know, I hate this function's name. I know it's not your fault at all, it's been there way before us, but `is25g` doesn't mean anything at all, it's annoying and confusing !